src/langtrace_python_sdk/extensions/langtrace_exporter.py

```diff
@@ -72,7 +76,7 @@ def export(self, spans: typing.Sequence[ReadableSpan]) -> SpanExportResult:
         Returns:
             The result of the export SUCCESS or FAILURE
         """
-        if not self.api_key:
+        if not self.api_key and not self.disable_logging:
```

Comment:
i am thinking a bit about this, 

this approach will work fine for now, but it's adding some tech debt to the langtrace and langtrace exporter
i would say a more scalable approach is the following 
1. make the disable logging as an enviroment variable not an option to be passed
2. abstract all Prints with `Fore` into a seperate function inside utils, that recieves 2 arguments which are the message and the color
3. inside the abstracted method just check for the env variable if it's set then return and do not print other wise print normally


```suggestion
        def log(message, color):
           disable_logging = os.environ['DISABLE_LOGGING']
           if disable_logging:
               return
           print(color, message, FORE.reset)
```
what do you think?